### PR TITLE
Refactoring, fixes for 80, 81

### PR DESCRIPTION
**Pull request by smecsia**

### CODE REVIEW

File: src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java [hunk 1]
```diff
@@ -18,6 +18,9 @@
     public void testDownloads() throws IOException {
         IArtifactStore artifactStore = new PostgresArtifactStoreBuilder().defaults(Command.Postgres).build();
 
+        if(true){
```
Comment (BorzdeG) [inline src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java]: OMG

Comment (smecsia) [inline src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java]: Yup. OMG. This is temporary. Rule is "Don't commit after 12pm:)"

Comment (BorzdeG) [inline src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java]: Use the @Ignore annotation from JUnit - so in the logs you will see the disabled tests

Comment (BorzdeG) [inline src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java]: demo: https://github.com/BorzdeG/ru.itbasis.gradle.plugins.postgresql-embedded-plugin/blob/master/src/test/groovy/ru/itbasis/gradle/plugins/postgresql/PostgresqlEmbeddedPluginTest.groovy#L68

Comment (smecsia) [inline src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java]: @BorzdeG I know about it, don't worry :).

**Pull request merged**